test/Bond.spec.ts

```diff
@@ -962,6 +1040,7 @@ describe("Bond", () => {
               bond.address,
               config.collateralTokenAmount
             );
+            bond.transfer(bondHolder.address, config.maxSupply);
```

Comment:
why doesn't this need awaited?

Reply:
I'm not sure. I added them to 12108b0c5dcbd3b19cfba5eec1b0af105a70ae05

I thought it might be due to auto-mine, but i disabled that and turned interval mining on and it seemed to work still.